src/runtime-rs/crates/hypervisor/src/utils.rs

```diff
@@ -25,3 +26,56 @@ pub fn get_child_threads(pid: u32) -> HashSet<u32> {
     }
     result
 }
+
+// get_virt_drive_name returns the disk name format for virtio-blk
+// Reference: https://github.com/torvalds/linux/blob/master/drivers/block/virtio_blk.c @c0aa3e0916d7e531e69b02e426f7162dfb1c6c0
+pub fn get_virt_drive_name(mut index: i32) -> Result<String> {
+    if index < 0 {
+        return Err(anyhow!("Index cannot be negative"));
+    }
+
+    // Prefix used for virtio-block devices
+    const PREFIX: &str = "vd";
+
+    // Refer to DISK_NAME_LEN: https://github.com/torvalds/linux/blob/08c521a2011ff492490aa9ed6cc574be4235ce2b/include/linux/genhd.h#L61
+    let disk_name_len = 32usize;
+    let base = 26i32;
+
+    let suff_len = disk_name_len - PREFIX.len();
+    let mut disk_letters = vec![0u8; suff_len];
+
+    let mut i = 0usize;
+    while i < suff_len && index >= 0 {
+        let letter: u8 = b'a' + (index % base) as u8;
+        disk_letters[i] = letter;
+        index = (index / base) - 1;
+        i += 1;
+    }
+    if index >= 0 {
+        return Err(anyhow!("Index not supported"));
+    }
```

Comment:
This chunk looks like it could be a separate function. That would allow it to be unit-tested with various values.